File: 11_November_Batch/C/reverseNumber.c
#include<stdio.h>
int reverse(int a){
    int sum=0,b;
    while(a!=0){
       b=a%10;
       sum=(sum*10)+b;
       a=a/10;
    }
    return sum;
}
int main(){
    int a , ans;
    scanf("%d",&a);
    ans = reverse(a);
    printf("%d",ans);
    return 0;
}

File: 4_August_Batch/Java/ReverseNumber.java
import java.util.*;
public class ReverseNumber{

    public static int reverse(int a){
        int sum=0;
        while(a!=0){
            int b=a%10;
            sum=(sum*10)+b;
            a/=10;
        }
        return sum;
    }
    public static void main(String args[]){
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter a number: ");
        int a = sc.nextInt();
        int x= reverse(a);
        System.out.println(x);

    }
}